README.md

```diff
@@ -1,57 +1,240 @@
-# Project Name
+<!--
+---
+name: Azure Functions C# Event Grid Blob Trigger using Azure Developer CLI
+description: This template repository contains an Azure Functions reference sample using the Blob trigger with Event Grid source type, written in C# (isolated process mode) and deployed to Azure using the Azure Developer CLI (azd). The sample uses managed identity and a virtual network to make sure deployment is secure by default.
+page_type: sample
+products:
+- azure
+- azure-functions
+- azure-blob-storage
+- azure-virtual-network
+- entra-id
+urlFragment: functions-quickstart-dotnet-azd-eventgrid-blob
+languages:
+- csharp
+- bicep
+- azdeveloper
+---
+-->
 
-(short, 1-3 sentenced, description of the project)
+# Azure Functions C# Event Grid Blob Trigger using Azure Developer CLI
+
+This template repository contains an Azure Functions reference sample using the Blob trigger with Event Grid source type, written in C# (isolated process mode) and deployed to Azure using the Azure Developer CLI (`azd`). When deployed to Azure the sample uses managed identity and a virtual network to make sure deployment is secure by default. You can opt out of a VNet being used in the sample by setting `SKIP_VNET` to true in the AZD parameters.
```

Comment:
Note when you do PR for updated bicep, this note on SKIP_VNET will change to VNET_ENABLED